service-down-page-config/remove-service-down-page.sh

```diff
@@ -1,14 +0,0 @@
-while true; do
```

Comment:
Should service-down-page-config/implement-service-down-page.sh be removed too?

Reply:
removed this not required